index.js

```diff
@@ -1 +1,11 @@
-// Code your solution here
+const findMatching = (drivers,name) => {
+  return drivers.filter(driver => driver.toLowerCase() === name.toLowerCase());
+}
+
+const fuzzyMatch = (drivers, name) => {
+  return drivers.filter(driver => driver.substring(0,1) === name.substring(0,1));
+}
```

Comment:
So this works, and passes the test, but not for all examples (if the name provided was "Ste", this would provide a false positive for "Stove" since it's only looking at the first letter of either). Take a peak at the `startsWith` function. You don't need to make changes, but if you have extra time, play around with it. Thanks!

```suggestion
const fuzzyMatch = (drivers, name) => {
  return drivers.filter(driver => driver.substring(0,1) === name.substring(0,1));
}
```